tests/integration/test_frbr_process.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from datetime import datetime, timedelta, timezone
+from processes import ClassifyProcess
+from model import Record
+
+
+def test_frbr_process(db_manager, seed_unclassified_record):
+    record_uuid = seed_unclassified_record
```

Comment:
If we change the above fixture to `test_unfrbrized_record_uuid` this line is not needed.